include/GroverSimulator.hpp

```diff
@@ -33,6 +33,7 @@ class GroverSimulator : public Simulator {
     }
 
     std::map<std::string, unsigned int> Simulate(unsigned int shots) override;
+    std::map<std::string, double> StochSimulate() override {};
```

Comment:
This should not be here, if it is not implemented.
Except if your intention is to exclude stochastic simulation from all these special simulators.
If this is the intention, then the function should at least return something because it is non-void.

Reply:
I currently don't plan on including these special simulators in the stochastic simulator. I added a return value to the function.